syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.epam.vshautsova.grpc";

package com.epam.vshautsova.grpc;

service MessageService {
  rpc GetMessage(MessageRequest) returns (MessageResponse) {}
}

message MessageRequest {
  string message = 1;
}

message MessageResponse {
  string message = 1;
}